### DISCUSSION TITLE
Persistent Storage

### DISCUSSION THREAD

**Discussion by KarateKitty** [linuxmint/discussions#341] [Ideas]
Can there be a separate Linux Mint that runs very well off a USB with persistent storage like tails OS. I do my work on multiple computers and it would be nice to keep my files while using a separate USB rather than sticking with the installation media, were everything clears after it is turned off.

**Comment by silviu-domotips-ro** [comment]
Came here for the same thing.  From time to time, i find myself using someone else's computer for unplanned stuff. I boot off an USB stick with mint and all good. 
But it would be usefull to have some kind of persistent storage, like casper filesystem in a file. It would be nice if the live iso would have an option/tool to create such a storage from within.

Thanks!
